statsd/src/main/java/com/avast/metrics/statsd/StatsDGauge.java

```diff
@@ -75,4 +75,8 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public void init() {
+        client.recordGaugeValue(name, 0);
```

Comment:
1) Are you sure it's really needed for gauges? The values are actively reported in a scheduled task, see `newGauge()`.

```
final ScheduledFuture<?> scheduled = scheduler.scheduleAtFixedRate(gauge::send, 0, gaugeSendPeriod.toMillis(), TimeUnit.MILLISECONDS);
```

2) Is there any wire difference between sending long and double? Are they compatible? May it break something? See `sendLongValue()` and `sendDoubleValue()` in this class.